common/djangoapps/student/urls.py

```diff
@@ -18,7 +18,6 @@
     url(r'^accounts/disable_account_ajax$', views.disable_account_ajax, name="disable_account_ajax"),
     url(r'^accounts/manage_user_standing', views.manage_user_standing, name='manage_user_standing'),
 
-    url(r'^change_setting$', views.change_setting, name='change_setting'),
```

Comment:
**Note to Reviewers:** I double-checked in New Relic that this API has been called 0 times since 6 weeks ago on Stage, Prod, and Edge.  I believe it's a much older endpoint since `user_api` was created.

I'd like to move the remaining (email and activation) account-related views to `user_api` in a subsequent PR.  Then, `student` will only contain functionality for the student `dashboard` and `roles`.